Makefile.devcontainer

```diff
@@ -83,7 +83,7 @@ run-race: install-race _run     ## Run pmm-managed with race detector.
 # 	bin/pmm-managed.test -test.coverprofile=cover.out -test.run=TestMainCover $(RUN_FLAGS)
 
 psql:                           ## Open psql shell.
-	env PGPASSWORD=pmm-managed psql -U pmm-managed pmm-managed-dev
```

Comment:
I'm not sure about it. There are two databases: one for unit tests, another for running instance. I guess we should just have two targets

Reply:
done, sorry for the delay completely forgot about this PR